rmw_implementation/src/functions.cpp

```diff
@@ -679,21 +688,19 @@ void prefetch_symbols(void)
   GET_SYMBOL(rmw_get_subscriptions_info_by_topic)
 }
 
-void * symbol_rmw_init = nullptr;
-
 rmw_ret_t
 rmw_init(const rmw_init_options_t * options, rmw_context_t * context)
 {
   prefetch_symbols();
-  if (!symbol_rmw_init) {
-    symbol_rmw_init = get_symbol("rmw_init");
+  if (!g_symbol_rmw_init) {
+    g_symbol_rmw_init = get_symbol("rmw_init");
```

Comment:
@cottsay meta: you may want to revisit `CALL_SYMBOL` and `RMW_INTERFACE_FN` macros as well for global variable naming updates.

Reply:
I just noticed that like 10 mins ago. I'm going to revert this part of the change - maybe we can follow up sometime to clean up the whole pattern at once.